app/services/airbnb_service.rb

```diff
@@ -0,0 +1,27 @@
+class AirbnbService
+
+  def initialize(location)
+    @location = location
+    @conn = Faraday.new(url: "https://www.airbnb.com/") do |faraday|
+      faraday.headers["X-API-KEY"] = ENV["airbnb_key"]
+      faraday.adapter Faraday.default_adapter
+    end
+  end
+
+  def find_properties
```

Comment:
If we have time we could try to break this long link down with some string interpolation but that would be a very final refactor.